clients/cloud/ruby/c_cloud.rb

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/ruby
+#
+# Copyright 2018 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# =============================================================================
+#
+# Helper module
+#
+# =============================================================================
+
+require 'optparse'
+require 'kafka'
+
+#
+# Helper class for working with Confluent Cloud
+#
+class CCloud
+  attr_reader :kafka
+
+  def initialize
+    # Parse arguments, load the CCloud config and initialize the Kafka client
+    @args = parse_args!
+    @config = load_config(@args[:config])
+    @kafka = Kafka.new(
```

Comment:
What is telling the client to connect with SSL?

Reply:
the `ssl_ca_certs_from_system` arg

From the zendesk docs:

> Authentication using SASL
Kafka has support for using SASL to authenticate clients. Currently GSSAPI, SCRAM and PLAIN mechanisms are supported by ruby-kafka.

> NOTE: With SASL for authentication, it is highly recommended to use SSL encryption. The default behavior of ruby-kafka enforces you to use SSL and you need to configure SSL encryption by passing ssl_ca_cert or enabling ssl_ca_certs_from_system. However, this strict SSL mode check can be disabled by setting sasl_over_ssl to false while initializing the client.